### PR TITLE
Convert Ui Account Package to Js

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-ui-account/client/accountProfile.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+			title: t('Please_enter_your_password'),
+			text: t('For_your_security_you_must_enter_your_current_password_to_continue'),
+			type: 'input',
+			inputType: 'password',
+			showCancelButton: true,
+			closeOnConfirm: false,
+			confirmButtonText: t('Save'),
+			cancelButtonText: t('Cancel')
+		}, () => {
+			return function(typedPassword) {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/accountProfile.js]: you are returning a function that never will be executed.

File: packages/rocketchat-ui-account/client/accountProfile.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+				title: t('Are_you_sure_you_want_to_delete_your_account'),
+				text: t('If_you_are_sure_type_in_your_password'),
+				type: 'input',
+				inputType: 'password',
+				showCancelButton: true,
+				closeOnConfirm: false,
+				confirmButtonText: t('Delete'),
+				cancelButtonText: t('Cancel')
+			}, () => {
+				return function(typedPassword) {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/accountProfile.js]: you are returning a function that never will be executed.

File: packages/rocketchat-ui-account/client/accountProfile.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+			return swal({
+				title: t('Are_you_sure_you_want_to_delete_your_account'),
+				text: t('If_you_are_sure_type_in_your_username'),
+				type: 'input',
+				showCancelButton: true,
+				closeOnConfirm: false,
+				confirmButtonText: t('Delete'),
+				cancelButtonText: t('Cancel')
+			}, () => {
+				return function(deleteConfirmation) {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/accountProfile.js]: you are returning a function that never will be executed.

File: packages/rocketchat-ui-account/client/accountProfile.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+			});
+		}
+	},
+	'click #resend-verification-email'(e) {
+		const user = Meteor.user();
+		e.preventDefault();
+		e.currentTarget.innerHTML = `${ e.currentTarget.innerHTML } ...`;
+		e.currentTarget.disabled = true;
+		return Meteor.call('sendConfirmationEmail', user.emails && user.emails[0] && user.emails[0].address(() => {
+			return function(error, results) {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/accountProfile.js]: you are returning a function that never will be executed.

File: packages/rocketchat-ui-account/client/avatar/avatar.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Template.avatar.helpers({
+	imageUrl() {
+		let username = this.username;
+		if ((username == null) && (this.userId != null)) {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/avatar/avatar.js]: remove `(` ... `)`

File: packages/rocketchat-ui-account/client/avatar/prompt.js [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+					return RocketChat.callbacks.run('userAvatarSet', tmpService);
+				}
+			});
+		}
+	},
+	'click .login-with-service'(event, template) {
+		const loginWithService = `loginWith${ _.capitalize(this) }`;
+		const serviceConfig = {};
+		return Meteor[loginWithService](serviceConfig, function(error) {
+			if ((error && error.error) === 'github-no-public-email') {
```
Comment (ggazzo) [inline packages/rocketchat-ui-account/client/avatar/prompt.js]:
```javascript
if (error) {

  if(error.error === 'github-no-public-email') {
    return alert(t('github_no_public_email'));
  
  }
  console.log(error)
  return toastr.error(error.message);
}
```

**Comment by MartinSchoeler** [comment]
I will Improve some things pointed by @ggazzo

**Comment by MartinSchoeler** [comment]
@ggazzo or @rodrigok could you do a review?

**Pull request merged**